app/config/locales/en.yml

```
@@ -1,14 +1,15 @@
 ---
 en:
   applicant_mailer:
-    caseworker_summary_email:
-      body: Applicant has submitted their income verification. Please find the attached PDF.
-      subject: 'Applicant Income Verification: %{case_number}'
     invitation_email:
-      body: A government agency has requested that you verify your income in connection with your application for benefits. Please click this link to begin the process.
+      body_html: We need to verify your income as part of your benefits application, as requested by %{agency_acronym}. This process is unique to you, so please do not share this invitation with anyone else. <strong>You have until %{deadline}, to complete your verification.</strong> Otherwise, you'll need to request a new invitation.
```

I know the design had it, but, the comma after the deadline date is not grammatically correct.

```suggestion
      body_html: We need to verify your income as part of your benefits application, as requested by %{agency_acronym}. This process is unique to you, so please do not share this invitation with anyone else. <strong>You have until %{deadline} to complete your verification.</strong> Otherwise, you'll need to request a new invitation.
```